blueprints/modular/intro.yaml

```
@@ -0,0 +1,58 @@
+title: Intro
+'@extends': default
+
+form:
+  fields:
+    tabs:
+      fields:
+        buttons:
+          type: tab
+          title: Intro
+          fields:
+
+            header.links:
+              type: list
+              label: Intro Links
+              style: horizontal
+              fields:
+
+                .link:
+                  type: text
+                  label: URL
+
+                .icon:
+                  type: text
+                  label: Font Awesome Icon Classes
+                  description: For example, "fas fa-envelope"
+
+            header.use_footer_links:
+              type: toggle
+              label: Use Footer Links
+              highlight: 1
+              description: If activated, the Footer Links defined in the theme configuration will be shown with the links defined here.
+              options:
+                1: Yes
+                0: No
+              validate:
+                type: bool
+
+            header.image:
+              type: filepicker
+              label: Custom Intro Image
+              folder: 'self@'
+              preview_images: true
+              description: Choose from an image uploaded to the Page Media section of the Content tab to set the Intro image.
```

We can change this one as well since it'll be in the Content tab.
```suggestion
              description: Choose from an image uploaded to the Page Media section to set the Intro image.
```